app/templates/create.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 rounded bg-white border border-primary" id="createForm" method="put">
+    {{ create_form.hidden_tag() }}
+
+    {% for field in create_form %}
+    {% if field.type == "StringField" or field.type == "DateTimeLocalField" %}
+    <div class="row my-3 gy-2">
+      {{ field.label(class="col-form-label col-2") }}
```

Comment:
Formatting still weird on mobile: col 2 is there regardless of size still (use either col-sm-2 or just col).
Consider for such a big form only using margin top - at least personally i feel theres too much space between input elements.